### ISSUE TITLE
Wifi not connecting

### ISSUE THREAD

**Issue by mhamilt** [s1858422/Ambient-Light-to-Adafruit-IO#1]
In its previous state the code did not compile due to multiple errors

In its current state the code compiles but does not connect to Wifi.

**Comment by mhamilt** [comment]
- Have you tried a hotspot from a phone?
- Are you 100% sure the SSID and password are correct?
- Has it ever successfully connected to WiFi?
  - if so, go back and run an example sketch. Is it still working?

**Comment by s1858422** [comment]
Yeah, so I have been able to connect to wifi before with this SSID and Password using a different sketch. Do you think it would be beneficial for me to open up a new blank document and rewrite out the code and see if I get anywhere with that?

**Comment by mhamilt** [comment]
Definitely, don’t even need to rewrite, just use the example provided by WiFiNINA library

**Comment by s1858422** [comment]
Ah yeah, that sounds like a plan, will try that, I always forget about the library examples

**Comment by s1858422** [comment]
Hi Matthew, I have updated the code and I have managed to connect it to wifi and in turn to MQTT, this issue is everytime I uncomment out the contents of the void loop, it still comes up with the "connectToWifi is not declared in the scope" I have searched the issue and can't seem to find anything that might be causing. I don't really understand why it works fine until I try to publish stuff to MQTT, then it throws up that error again.

**Comment by mhamilt** [comment]
> everytime I uncomment out the contents of the void loop, it still comes up with the "connectToWifi is not declared in the scope"

which line in particular does it start happening when you uncomment it

**Comment by mhamilt** [comment]
Where is the closing `}` for the `loop`?

https://github.com/s1858422/Ambient-Light-to-Adafruit-IO/blob/51b1609ed751a520857f81f3e6fb7dd9eb3cdd0e/Ambient_light_sensor/Ambient_light_sensor.ino#L23-L55

**Comment by s1858422** [comment]
Hi Matthew, sorry that is my bad, feel a little silly now, it is up and running now. Sorry, should have checked more thoroughly before messaging you.
